image-definitions/README.md

```diff
@@ -52,7 +55,7 @@ Initial setup of image definitions was already peformed. If required, initial se
 setup.sh .
```

Comment:
the `### Setup` and `### Publish` sections feel a bit confusing now that this uses patches instead of committing the entire upstream info.  Setup says "Initial setup ... was already performed" which I think would be taken as "I dont need to do it again", which was very applicable when we stored the upstream files but now isn't quite the same.  Before, a user could just `git clone; follow publish instructions`, but now need to pull the repo and apply a patch before publishing

Reply:
I see your point. Will change.

Reply:
Done.